docs/Horizon/Horizon-overview.md

```diff
@@ -0,0 +1,73 @@
+# Horizon RWA Instance
+
+## Context
+- Horizon is an initiative by Aave Labs focused on creating Real-World Asset (RWA) products tailored for institutions.
+- Horizon will launch a licensed, separate instance of the Aave Protocol (initially a fork of v3.3) to accommodate regulatory compliance requirements associated with RWA products.
+- Horizon Instance will have a dual role setup with responsibilities split between Aave DAO (Operational role) and Aave Labs (Executive role).
+ 
+## Overview
+The Horizon Instance will introduce permissioned (RWA) assets. The Aave Pool will remain the same, except that RWA assets can be used as collateral in order to borrow stablecoins. Permissioning occurs at the asset level, with each RWA token issuer enforcing asset-specific restrictions directly into their ERC-20 token. The Aave Pool is agnostic to each specific RWA implementation and its asset-level permissioning.
+
+From an Issuer perspective, aTokens are an extension of the RWA tokens, which are securities. The aTokens will signify ownership of the supplied underlying RWA Token. To accomodate edge cases, a protocol-wide aToken Transfer Admin is also added, allowing Issuers the ability to forcibly transfer aTokens on behalf of end users (without needing approval). These transfers will still enforce collateralization and health factor requirements as in existing Aave peer-to-peer aToken transfers.
+
+There is permissionless supply of USDC to the Horizon Instance which can earn yield. GHO will also be listed in Horizon as a standard, non-mintable stablecoin. Borrowable stablecoins will not be used as collateral. 
+
+Liquidations will be limited to entities already allowlisted to hold the underlying RWA collateral asset.
+
+## Expected Code/Functionality Changes
+
+### RWA Asset (Collateral Asset)
+A new RWA-specific aToken contract (`RwaAToken`) has been developed, which prevents internal/external (peer-to-peer) transfers and allowance-related methods for users. It also incorporates an authorized ATokenAdmin role (`ATOKEN_ADMIN_ROLE`) which can transfer aTokens on behalf of users within collateralization/health factor constraints.
```

Comment:
I d give a preamble here instead, saying that RWA assets can be listed by using a new developed aToken contract which restricts the functionality of the RWA underlying asset in the Aave Pool.
We explain 1 by 1 below in the list